assets/js/watch.js

```diff
@@ -195,3 +195,15 @@ addEventListener('load', function (e) {
         comments.innerHTML = '';
     }
 });
+
+addEventListener("DOMContentLoaded", () => {
```

Comment:
I don't remember if internet explorer is still a supported browser or not but arrow functions aren't supported in IE. So if it is still supported it will need to be changed. https://caniuse.com/arrow-functions

```suggestion
addEventListener("DOMContentLoaded", function() {
```